testsuite/pytests/mpi/2/test_issue_576.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+#
+# test_issue_576.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import nest
+import sys
+import unittest
+
+HAVE_GSL = nest.ll_api.sli_func("statusdict/have_gsl ::")
+
+
+@unittest.skipIf(nest.NumProcesses() < 2, "Requires >= 2 MPI process")
```

Comment:
For consistency. I chose to flip the double quotes because single quotes are used the most.
```suggestion
HAVE_GSL = nest.ll_api.sli_func('statusdict/have_gsl ::')


@unittest.skipIf(nest.NumProcesses() < 2, 'Requires >= 2 MPI process')
```